app/src/main/res/values-pt-rBR/strings.xml

```diff
@@ -0,0 +1,169 @@
+    <string name="tip_urlbar">Você pode digitar o nome de host ou endereço de IP.\n\'vnc://\' URIs também são suportados.</string>
+    <string name="tip_discovery">A descoberta pode encontrar automaticamente os servidores suportados\nexecutados na sua rede.</string>
+
+    <!-- Accessibility descriptions -->
+    <string name="desc_navigation_btn">Abrir a aba de navegação</string>
+    <string name="desc_settings_btn">Abrir configurações</string>
+    <string name="desc_saved_servers_tab">Servidores salvos</string>
+    <string name="desc_discovered_servers_tab">Servidores encontrados</string>
+    <string name="desc_add_new_server_btn">Adicionar um novo servidor</string>
+    <string name="desc_save_discovered_server_btn">Salvar este servidor</string>
+    <string name="desc_discovery_btn">Iniciar a descoberta de servidores</string>
+    <string name="desc_discovery_running">Descobrindo servidores na sua rede</string>
+    <string name="desc_keyboard_btn">Mostrar teclado</string>
+    <string name="desc_virtual_keys_btn">Mostrar teclas virtuais</string>
+    <string name="desc_reset_zoom_btn">Resetar o zoom para 100%</string>
+    <string name="desc_retry_connection_btn">Tente conectar novamente</string>
+    <string name="desc_back_btn">Voltar</string>
+    <string name="desc_clear_url_btn">Remover a URL</string>
+    <string name="desc_server_url">Digite o endereço do servidor VNC</string>
+    <string name="desc_help_btn">Mostrar ajuda</string>
+    <string name="desc_discovered_list">Lista de servidores descobertos automaticamente</string>
+    <string name="desc_saved_servers_list">Lista de servidores salvos</string>
+
+    <!-- Preference -->
+    <string name="pref_ui">Interface</string>
+    <string name="pref_ui_summary">Aspecto</string>
+    <string name="pref_viewer">Visualizador</string>
+    <string name="pref_viewer_summary">Imagem-na-imagem, Zoom</string>
+    <string name="pref_input">Entrada</string>
+    <string name="pref_input_summary">Gestos, Mouse, Teclas virtuais</string>
+    <string name="pref_servers">Servidores</string>
+    <string name="pref_server_summary">Descoberta, Credenciais</string>
+    <string name="pref_tools">Ferramentas</string>
+    <string name="pref_tools_summary">Importar/Exportar, Depuração</string>
+    <string name="pref_experimental">Experimental</string>
+    <string name="pref_experimental_summary">Mais recursos!</string>
+
+    <string name="pref_gesture_action_none">Nenhum</string>
+    <string name="pref_gesture_action_left_click">Clique esquerdo</string>
+    <string name="pref_gesture_action_middle_click">Clique do meio</string>
+    <string name="pref_gesture_action_right_click">Clique direito</string>
+    <string name="pref_gesture_action_double_click">Clique duplo</string>
+    <string name="pref_gesture_action_move_pointer">Mover ponteiro</string>
+    <string name="pref_gesture_action_pan">Focar no quadro local</string>
```

Comment:
This seems slightly off to me (correct me if I am wrong here). 'Focar' seams to mean 'Focus', but in English version 'Pan' is used in the sense of 'move across'. It 'moves' the visible area of the frame.

Online tools translate 'Pan local frame' to 'Pan quadro local', but you may have abetter idea about it.

Reply:
sorry I was kinda busy, I changed that entry